source/scripts/model/Monster.js

```diff
@@ -100,6 +100,49 @@ export default class Monster {
         this.position.x += movement.x
         this.position.y += movement.y
     }
+    pursuit(magnitude) {
```

Comment:
Can you change this method so instead of mutating the monster's movement, it just returns something like `{x: +1}` that we can pass to the `move` method?